scene/3d/light_3d.cpp

```diff
@@ -176,6 +176,11 @@ Ref<Texture2D> Light3D::get_projector() const {
 	return projector;
 }
 
+void Light3D::owner_changed_notify() {
+	// For cases where owner changes _after_ entering tree (as example, editor editing).
```

Comment:
Sounds like a comment that should go in `node.cpp/h` to clarify what this core API is for.

Reply:
I could add one there (though the function is quite self explanatory by name), but this comment is meant specifically for these nodes, because the editor-only lights need to check the owner to verify they are in the right scene and hence see that it must be enabled or disabled.